tools/dynamic_framework_slicer/dynamic_framework_slicer.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash 
+set -eu
+
+BINARIES+=()
+while [[ $# -gt 0 ]]; do
+  arg="$1"
+  case $arg in
+      --in)
+      readonly IN="$2"
+      shift
+      ;;
+      --out)
+      readonly OUT="$2"
+      shift
+      ;;
+      --realpath)
```

Comment:
Why do you need realpath?

Reply:
No clue. This was something in the skeleton that @sergiocampama  provided. I am happy to remove if it isn't needed (it obviously isn't used in the script, but that could be a bug?).

Reply:
Likely just because he copied something quickly to help get things going for you.  If it isn't needed, drop it.

Reply:
Roger. Will remove.